### DISCUSSION TITLE
PatchTool 6.4 5/26 Bug Fixes

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#347] [Announcements]
修正上版本的指令碼錯誤、更新大廳文字。

檔案：

- FFXIVChnTextPatch-20230526.rar：本體。一般而言，下載這個即可。

選用：

- [font-custom-4k-smaller-20230131.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.0.1/font-custom-4k-smaller-20230131.rar)：高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)透過重複使用字體字型縮小檔案體積，使用Noto Sans。
- [font-custom-20230111.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.72.2/font-custom-20230111.rar)：舊高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)所有大小字體皆分開繪製，使用Noto Sans和Noto Serif。
- rawexd-zhcn-20230526.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [FFXIVChnTextPatch.jar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.1.0.0/FFXIVChnTextPatch.jar)：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- [backup-20230523.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.1.0.0/backup-20230523.rar)：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

※若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。更多資訊可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v3.1.0.2'>PatchTool 6.4 5/26 Bug Fixes</a>.</em>

**Comment by jc-ff14** [comment]
回報：角色在漢化過後, 接了2.0的水晶塔的起始任務："古代亞拉戈文明的遺產", 
並去指定地點 (NPC: 拉姆布鲁斯-摩杜纳 X:30.4, Y:12.1) 交任務, 
抵達NPC所在之處時, 遊戲Crash, 而且無法再登入遊戲, 進選單就Crash, 
試過還原/再重新漢化/換角色/換服務器登入...各種方法均無效, 
最後只能移除遊戲重新安裝.

以上狀況是由兩台電腦/兩個不同帳號/均完成漢化後...所有設定條件相同之下, 
同一時間一起去解相同任務後, 出現一模一樣的Crash狀況, 
且兩台都嘗試過各種排除方式均無法修復, 都只能移除遊戲重新安裝, 
兩台電腦重新安裝並重新漢化號之後就沒有Crash, 
但是重新安裝後就把角色移開, 不再停留在該處後, 才重新漢化的!!

此外重新漢化之後就沒有再嘗試過去那個地點, 
因為重新安裝很麻煩, 希望製作人幫忙處理這個問題, 萬分感謝!!

**Reply by GpointChen** [reply]
你是角色一靠過去就崩潰嗎？沒有對話或做其他事情？

**Reply by GpointChen** [reply]
我剛測試了一下我過去以及跟目標NPC對話都沒事，可以確認一下是不是最新版的漢化（v3.1.0.2）？

另外你說到的「還原後也會crash」比較奇怪，因為還原是直接換成你原來的檔案，不確定你是否原本檔案有損毀所致。

**Comment by jc-ff14** [comment]
對，重點是：兩台電腦同時同秒，飛過去一到當地就crash,
還原後也是一開遊戲列表就Crash，兩台電腦一北一南（南部的是我朋友的）我們兩台都有同樣的狀況，一摸一樣、同時發生

GpointChen ***@***.***>於 2023年5月29日 週一，下午7:37寫道：

**Reply by GpointChen** [reply]
遊戲列表是指選好角色後進遊戲還是哪一步？

**Comment by HumHumHum3** [comment]
抱歉想請問一下，按照步驟做結果無法漢化怎麼辦，謝謝

**Comment by Bos-hunting** [comment]
你好，6月6日中國服好像有更新6.31的內容，希望會有內容的兼容版（例如6.3絕本的漢化以及絕本稱號之類的），感謝
